index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Bowling</title>
+  <link href='public/css/style.css' rel='stylesheet' type='text/css'>
+  <script src='src/jquery-3.2.1.min.js'></script>
+  <script src='src/game.js'></script>
+  <script src='src/interface.js'></script>
+</head>
+
+<body>
+  <div id='title' class='title'><h1>Bowling!</h1></div>
+  <div id='buttons' class='buttons'>
+    <p>Enter on your roll</p>
+    <button class="rollScore"> 0 </button>
+    <button class="rollScore"> 1 </button>
+    <button class="rollScore"> 2 </button>
+    <button class="rollScore"> 3 </button>
+    <button class="rollScore"> 4 </button>
+    <button class="rollScore"> 5 </button>
+    <button class="rollScore"> 6 </button>
+    <button class="rollScore"> 7 </button>
+    <button class="rollScore"> 8 </button>
+    <button class="rollScore"> 9 </button>
+    <button class="rollScore"> 10 </button>
+    <br>
+  </div><br>
+
+  <div id='scoresheet'>
+    <table id='scoresheetTable' class='scoresheet' cellpadding='1' cellspacing='0'>
+      <tr>
```

Comment:
An alternative to treating this as a table is to use a series of divs. If you refactor you game code to use frame classes such that you have an array of frames, you could write a method here like:

```
function buildGameCard() {
    $container = $('#score-card-container');
    game.getFrames().forEach((frame) => {
        var frameView = buildFrameView(frame);
        $container.append(frameView);
    });
}

function buildFrameView(frame) {
    var frameView = $('
        <div class='frame'>
        <span class='roll1'></span><span class='roll2'>
        <span class='score'></score>
        </div>');
    frameView.find('.roll1').text(frames.getFirstRoll());
    frameView.find('.roll2').text(frames.getSecondRoll());
    frameView.find('.score').text(frames.getScore());
    return frameView;
}
```